torch/quantization/_numeric_suite.py

```diff
@@ -51,6 +64,15 @@ def compare_weights(float_dict, quantized_dict):
             weight_dict[key] = {}
             weight_dict[key]["float"] = float_dict[match_key]
             weight_dict[key]["quantized"] = quantized_dict[key]
+            continue
+
+        # For matching "fc.weight" and "fc._packed_params._packed_params"
```

Comment:
You will need to look at the version of the state dict to figure out what needs to be done here. State dicts will change over time, so having  a check for the version number is a good idea. This is available at the level of each module's state dict.

Reply:
Could you give an example how to check the state dict version?

Reply:
The state dict has a field called _version for each module, an example is at:
https://github.com/pytorch/pytorch/blob/master/torch/nn/quantized/modules/linear.py#L11
The main code in module that does state dict serialization is at: 
https://github.com/pytorch/pytorch/blob/master/torch/nn/modules/module.py#L851